docs/library/README.md

```
@@ -0,0 +1,398 @@
+### Installation
+
+```bash
+go get github.com/BumpyClock/digests-api/digests-lib
+```
+
+### Basic Usage
+
+```go
+package main
+
+import (
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)